setup.py

```diff
@@ -90,6 +90,7 @@
         ],
         "invenio_base.api_blueprints": [
             "cds_ils_patron_loans = cds_ils.patrons.views:create_patron_loans_blueprint",
+            "cds_ils_legacy = cds_ils.literature.views:legacy_blueprint",
```

Comment:
missing to add the minter (see invenio-app-ils)
```
"invenio_pidstore.minters": [
```

Reply:
added below